test/password.cpp

```diff
@@ -0,0 +1,22 @@
+#include "catch.hpp"
+
+#include <password.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("read_password()", "[password]") {
```

Comment:
It's better to give tests descriptive explanations, so when a test fails, it's clear what functionality was broken. In this case, the description might say "read_password() returns the first line of the file".